server/server.go

```diff
@@ -68,26 +71,37 @@ func StartServer(ctx context.Context, cfg *config.Config) error {
 		if err != nil {
 			return err
 		}
-		if len(cfg.Service.GatewayAddresses) > 0 || cfg.Service.DisableBroadcast {
+		gtwManager, _ := gateway.NewGatewayManager(ctx, cfg.Service.GatewayAddresses)
+		if len(gtwManager.Connections()) > 0 {
```

Comment:
If the check for the required number of connections already happens in `NewGatewayManager` this could simplify to:

```golang
gtwManager, err := gateway.NewGatewayManager(ctx, ctg.Service.GateWayAddress, 1)
if err == nil {
   // no error and enough connections
} else {
    // handle error
}
```

Now errors are just ignored.